qa-pipelines/set-pipeline

```diff
@@ -155,7 +160,7 @@ fly \
     --non-interactive \
     --pipeline="${pipeline_name}" \
     --var pipeline-name="${pipeline_name}" \
-    --config="${pipeline_file}" \
+    --config=<(ruby generate.rb "${preset_file}" flags.yml) \
```

Comment:
Note that there's a separate manifest for the single-brain pipeline (see merged PR #240). Right now generate.rb doesn't account for this and will break when set-pipeline is run with the --single-brain flag. If the flag is set, generate.rb should generate a pipeline with one job for each brain test (see the block which is repeated with minor variations here )

Maybe this could share the template with the regular pipeline, but it might be easier to create a separate template for it.